src/components/Card/Card.scss

```diff
@@ -3,9 +3,9 @@ body {
     font-weight: 400;
     line-height: 1.5;
     letter-spacing: 0.01em;
-}
-
-button {
```

Comment:
No generic selector, encapsulate CSS in a component class. Recommendation:
In component use const COMP_NAME = 'card';

in className=`${COMP_NAME}-container`

in CSS use 
.card-container {
  &-button {....}
}